Makefile

```diff
@@ -8,8 +8,9 @@ ROLES := $(wildcard roles/*)
 PLUGIN_TYPES := $(filter-out __%,$(notdir $(wildcard plugins/*)))
 RUNTIME_YML := meta/runtime.yml
 METADATA := galaxy.yml LICENSE README.md $(RUNTIME_YML) requirements.txt changelogs/changelog.yaml CHANGELOG.rst bindep.txt PSF-license.txt meta/execution-environment.yml
+TESTDATA := Makefile pytest.ini $(shell find tests/ ! -type d ! -path '*/__pycache__/*' ! -path '*/test_playbooks/fixtures/*')
```

Comment:
the file blows up to 3.5MB if I do not exclude these fixtures

Reply:
Sounds a lot like `wildcard`, `notdir` and `filterout` to me... ;)

Reply:
yeah, but at what price? :)